src/index.html

```diff
@@ -8,6 +8,13 @@
   <link rel="stylesheet" href="./styles/main.css">
 </head>
 <body>
-  <h1>Antisnake</h1>
+  <div class="antisnake">
+    <div class="antisnake__cell">1</div>
```

Comment:
It seems convenient to me to add content with pseudo-element. In this case it's not necessary because you can easily automate it with Emmet, which is what you probably did:)